postgres-native-tls/src/lib.rs

```diff
@@ -4,10 +4,13 @@
 //!
 //! ```no_run
 //! use native_tls::{Certificate, TlsConnector};
+//! # #[cfg(feature = "runtime")] {
```

Comment:
This is not valid syntax.

Reply:
This is valid, it just doesn't have the expected effect - it's a hidden line which only compiles the block when `feature = "runtime"` is set. But that feature will never be set for doctests, only for the library itself.

Reply:
```
error: expected item after attributes
 --> src/main.rs:1:1
  |
1 | #[cfg(feature = "runtime")] {
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: could not compile `playground` due to previous error
```

You cannot define bare blocks at module level.

Reply:
Oh, that was a stupid mistake, thanks.